### ISSUE TITLE
18238

### ISSUE THREAD

**Issue by gnakan** [gnakan/gd-cloud-docs#18]

**Comment by advaiya-main** [comment]
Moving back to our queue to address issues

**Comment by advaiya-main** [comment]
Edited as per the guidelines

**Comment by Carla-Johnson** [comment]
Proofread

**Comment by gnakan** [comment]
When running `sudo curl -L https://get.rvm.io | bash -s stable' get the message 'pg: Can't check signature: public key not found
Warning, RVM 1.26.0 introduces signed releases and automated check of signatures when GPG software found.
Assuming you trust Michal Papis import the mpapis public key (downloading the signatures).

GPG signature verification failed for '/home/vagrant/.rvm/archives/rvm-1.26.11.tgz' - 'https://github.com/rvm/rvm/releases/download/1.26.11/1.26.11.tar.gz.asc'!
try downloading the signatures:

    gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3

or if it fails:

    command curl -sSL https://rvm.io/mpapis.asc | gpg --import -

the key can be compared with:

    https://rvm.io/mpapis.asc
    https://keybase.io/mpapis'

**Comment by kharise** [comment]
@advaiya-main:
Started by testing on Ubuntu: Failed.

Please address these 2 changes.
1)  In "Set up a database connection", for step 2: clarify which line needs to be changed in the file. "password: " occurs twice. And do I set the password to the pw I'd created earlier? I assumed so but please clarify. 
2.In this file, type the following in the password section: 
password: mysql_root_password.

2) In that same section for step 4, I'm denied access.
OUTPUT:
kharise@server261-ubuntu:~/your_rails_app$ mysql -u root –p
ERROR 1045 (28000): Access denied for user 'root'@'localhost' (using password: NO)

Please sync, verify the steps for Ub and Deb, and send back once done.

**Comment by advaiya-main** [comment]
Corrected some of the commands.

**Comment by advaiya-main** [comment]
@kharise: Can you please prioritize this article by 10/20?

**Comment by kharise** [comment]
@advaiya-main: Yes, I've just tested this on Ubuntu and had to change " mysql -i -u root –p" to "mysql -u root -p" in order to get this to work. 
Is there a reason why I shouldn't do this? 
If not, I'll assume it's okay.I'm testing this on Debian now.

**Comment by kharise** [comment]
Passed testing on Ub/Deb
